src/prefect/config.toml

```diff
@@ -71,7 +71,7 @@ checkpointing = false
     [engine.executor]
 
     # the default executor, specified using a full path
-    default_class = "prefect.engine.executors.SynchronousExecutor"
+    default_class = "prefect.engine.executors.LocalExecutor"
```

Comment:
Out of curiosity, why this change?

Reply:
I'm willing to revert this, but with the known possibility for double execution in deep mapped pipelines, this felt safer.  I actually _thought_ we had already done this

Reply:
Totally fine, not asking for reversion was just curious why